Tasks/SonarQubePostTest/SonarQubePostTest.ps1

```diff
@@ -9,10 +9,16 @@ import-module "Microsoft.TeamFoundation.DistributedTask.Task.Common"
 ExitOnPRBuild
 
 . $PSScriptRoot/SonarQubePostTestImpl.ps1
-. $PSScriptRoot/SonarQubeBuildBreaker.ps1
 . $PSScriptRoot/PRCA/Orchestrator.ps1
 
+. $PSScriptRoot/SonarQubeMetrics.ps1
+. $PSScriptRoot/SonarQubeReportHandler.ps1
+. $PSScriptRoot/SonarQubeBuildBreaker.ps1
+
+
 InvokeMSBuildRunnerPostTest
-UploadSummaryMdReport
+# PRCA
 HandleCodeAnalysisReporting
+
+UploadSummaryMdReport
```

Comment:
I was originally thinking that doing PRCA first will give more time to SQ to perform the analysis. That being said, we only do PRCA in PR builds and in PR builds we rely on the legacy report. I'll switch it back